### PR TITLE
[Program: GCI] Create doc strings for Mentorship Relation API to improve Swagger documentation (Backend)

### LINKED ISSUE #404: Create doc strings for Mentorship Relation API to improve Swagger documentation
## Description

As a developer that uses the Swagger UI,
I need good API documentation on Swagger,
so that I can understand how to properly use the backend API.

The Mentorship Relation API resources don't have enough doc strings to reflect their usage or purpose on Swagger UI. By adding doc strings under the respective REST methods, you'll be updating the Swagger UI at the same time. E.g.: POST /login has a full description on Swagger.

Create doc strings for every resource/endpoint of Mentorship Relation API. **Hint:** the file where you might have to do these changes is [app/api/resources/mentorship_relation.py](https://github.com/systers/mentorship-backend/blob/develop/app/api/resources/mentorship_relation.py) file.
Be mindful to explain important parts of the logic behind the APIs, for example if an endpoint is only available to the creator of a request.

### Update [Required]
- [ ] Improve doc strings for every resource/endpoint of Mentorship Relation API. 

## Definition of Done
- [ ] All of the required items are completed.
- [x] Approval by 1 mentor.

## Estimation
2 hours

### PR DESCRIPTION
**Pull request by asawesome07**
### Description
Added doc strings to Resources/mentorship_relation.py for swagger UI.

Fixes #404 

### Type of Change:
- Code
- Quality Assurance
- User Interface

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?
Here are screenshots showcasing the doc strings.
Capture

d

### Checklist:
- [x] My PR follows the style guidelines of this project
- [x] I have performed a self-review of my own code or materials
- [x] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [x] Any dependent changes have been merged

**Code/Quality Assurance Only**
- [x] My changes generate no new warnings 
- [ ] My PR currently breaks something (fix or feature that would cause existing functionality to not work as expected)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been published in downstream modules

### CODE REVIEW

File: app/api/resources/mentorship_relation.py [hunk 1]
```diff
@@ -30,6 +30,13 @@ class SendRequest(Resource):
     def post(cls):
         """
         Creates a new mentorship relation request.
+
+        Required:
+        - Both users must be available for a mentorship relation and duration of relation must be between one
+        and six months.
```
Comment (sara-02) [inline app/api/resources/mentorship_relation.py]: between `one to six months.`

File: app/api/resources/mentorship_relation.py [hunk 2]
```diff
@@ -30,6 +30,13 @@ class SendRequest(Resource):
...
+
+        Returns:
+        - A message stating whether mentorship relation was successfully sent or not.
```
Comment (sara-02) [inline app/api/resources/mentorship_relation.py]: `whether a mentorship relation request was successfully initiated or not.`

**Review by sara-02 — COMMENTED** [review]
Hey @asawesome07 apart from the minor improvements in language, I suggest you also look at
https://www.python.org/dev/peps/pep-0257/ and http://google.github.io/styleguide/pyguide.html#381-docstrings for some standard ways of adding params, and return value in the docstring.

**Comment by isabelcosta** [comment]
Thank you so much for your contribution @asawesome07  during the GCI contest 🙌 
I'm closing this PR since we decided to merge PR #405

**Pull request closed without merging**